delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/config/AggregateConfig.scala

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.config
+
+import ch.epfl.bluebrain.nexus.sourcing.SnapshotStrategyConfig
+import ch.epfl.bluebrain.nexus.sourcing.processor.{EventSourceProcessorConfig, StopStrategyConfig}
+
+/**
+  * Aggregate configuration.
+  *
+  * @param stopStrategy     the stop strategy configuration for this aggregate [[EventDefinition]]
+  * @param snapshotStrategy the snapshot strategy configuration for this aggregate [[EventDefinition]]
+  * @param processor        the event source processor config
+  */
+//TODO: ported from service module, we might want to avoid this duplication
```

Comment:
I think it will be better do it soon

Reply:
Sure. We talked about it with bogdan this morning in the meeting. We'll do it when we find a common pattern to goup things